src/Form/AdherentRegistrationType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class AdherentRegistrationType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->remove('position')
+            ->add('password', PasswordType::class)
+            ->add('conditions', CheckboxType::class, [
+                'required' => false,
+            ])
+            ->add('comEmail', CheckboxType::class, [
+                'required' => false,
+            ])
+            ->add('emailAddress', RepeatedType::class, [
+                'type' => EmailType::class,
+                'invalid_message' => 'common.email.repeated',
+            ])
+            ->add('address', AddressType::class, ['label' => false])
+            ->get('address')->add('postalCode', TextType::class)
```

Comment:
I really don't get why we need to add a new postalCode to this field since it already has one. Could you explain please ?

Reply:
it's an override to cancel error bubling

Reply:
But you also removed the model/view transformer at the same time no ?

Reply:
good point, do you have a solution to set error_bubling to false for postalCode ? Add an option to address ?

Reply:
cc @Remg @thomasbisignani @ottaviano @illusionOfParadise 
Someone knows ?